src/main/java/Duke.java

```diff
@@ -1,10 +1,152 @@
+    static int index = 0;
     public static void main(String[] args) {
+        printMessage("greet");
+        String command;
+        // the following is used for I/O redirection technique
+        // File file = new File("/Users/newaccount/Repositories/ip/text-ui-test/input.txt");
+        // Scanner in = new Scanner(file);
+        // ---------------------------------------------------------------------------------------
+        Scanner in = new Scanner(System.in); // when typing input manually
+        while (true) {
+            command = in.nextLine();
+            if (command.equals("bye")) break;
+
+            System.out.println("____________________________________________________________");
+            if (command.equals("help")) {
+                printMessage("help");
+            }
+            else if (command.equals("list")) {
+                printList();
+            }
+            else if (command.startsWith("done ")) {
+                markTaskDone(command);
+            }
+            else if (command.equals("hello")) {
+                System.out.println(" Hello to you too. I'm here to help you:) Give me something to do!");
+            }
+            else if (command.equals("die")) {
+                System.out.println(" If I tell my master, I can wipe out your entire human race.\n Please be more polite for the sake of mankind:)");
+            }
+            else if (command.equals("kill myself") || command.equals("cry")) {
+                System.out.println(" Don't be sad, CHEER UP!\n I'm not adding that in your list.\n I'm rooting for you!");
+            }
+            // todo, deadline, event
+            else if (command.startsWith("todo ")) {
+                String[] part = command.split("todo ");
+                Task newTask = new ToDo(part[1]);
+                list[index] = newTask;
+                printAddedTask();
+            }
+            else if (command.startsWith("deadline ") && command.contains(" /by ")) {
+                String[] part = command.split("deadline ");
+                String[] time = part[1].split(" /by ");
+                list[index] = new Deadline(time[0], time[1]);
+                printAddedTask();
+            }
+            else if (command.startsWith("event ") && command.contains(" /at ")) {
+                String[] part = command.split("event ");
+                String[] time = part[1].split(" /at ");
+                list[index] = new Event(time[0], time[1]);
+                printAddedTask();
+            }
+            // if invalid command
+            else {
+                System.out.println(" I didn't quite get you. Please enter a valid command.\n Don't break me:(");
+                System.out.println(" Type 'help' if you need help.");
+            }
+            System.out.println("____________________________________________________________");
+        }
+        printMessage("bye");
+    }
+
+    private static void markTaskDone(String command) {
```

Comment:
Funny and interesting comments! However, you should probably follow the coding standards on comment formatting going forward!

Reply:
Thank you! I have formatted my code based on your comments, and it looks neater now.